.vscode/launch.json

```diff
@@ -15,7 +15,12 @@
             ],
             "outFiles": [
                 "${workspaceFolder}/out/**/*.js"
-            ]
+            ],
+            "preLaunchTask": "npm: compile",
```

Comment:
will this mean that the build will happen eveytime? if so, do we want this? would it be better to build seperately and keep launching as many times as you want?